### ISSUE TITLE
Turn off log formatting of messages printed by the hook

### ISSUE THREAD

**Issue by ayushtiku5** [flant/shell-operator#372]
I am using golang executable as a hook for the shell-operator. I want to use logging from inside the hook but it seems like anything that is printed to stdout/stderr from the hook gets log formatted by the shell operator inside the `msg` field.

```
{"binding":"kubernetes","event":"kubernetes","hook":"hook","level":"info","msg":"<msg>","output":"stderr","queue":"main","task":"HookRun","time":"2022-03-14T12:00:02Z"}
```

So, if we are logging something inside the hook, it gets nested inside the above `msg` field creating something like this

```
{"binding":"kubernetes","event":"kubernetes","hook":"hook","level":"info","msg":"{\"level\":\"info\",\"msg\":\"hello world\",\"time\":\"2022-03-14T12:00:02Z\"}","output":"stderr","queue":"main","task":"HookRun","time":"2022-03-14T12:00:02Z"}
```
**Expected behavior (what you expected to happen)**:
Is it possible to turn off the logging by the shell-operator? Or is it possible to not add the log messages printed by the hook inside another log message of shell-operator?

**Steps to reproduce**:
Here is the code of the hook
```
package main

import (
	"flag"
	"fmt"

	log "github.com/sirupsen/logrus"
)

func main() {
	var config bool
	flag.BoolVar(&config, "config", false, "")
	flag.Parse()

	log.SetFormatter(&log.JSONFormatter{})

	if config {
		yaml :=
`configVersion: v1
kubernetes:
- apiVersion: v1
  kind: ConfigMap
  executeHookOnEvent: ["Added", "Modified", "Deleted"]
  nameSelector:
   matchNames:
    - test_cm
  namespace:
   nameSelector:
    matchNames: ["test_ns"]`
		fmt.Println(yaml)
	} else {
		log.Info("hello world")
	}
}
```

**Comment by diafour** [comment]
It's interesting, we never consider structured logging from hooks. I think we can implement some settings to configure stdout handling in the future. Right now there is no quick workaround to print JSON messages from hook without embedding into JSON messages from shell-operator.
I can offer you to use TextFormatter. And also take a look at [Go hooks](https://github.com/flant/addon-operator/tree/main/examples/700-go-hook) in addon-operator: you can drop helm and use global hooks to get shell-operator experience.

**Comment by diafour** [comment]
New flag was added by #383 , available in 1.0.10
